src/gridstack-engine.ts

```diff
@@ -345,7 +345,7 @@ export class GridStackEngine {
 
     if (node.maxW) { node.w = Math.min(node.w, node.maxW); }
     if (node.maxH) { node.h = Math.min(node.h, node.maxH); }
-    if (node.minW) { node.w = Math.max(node.w, node.minW); }
+    if (node.minW && node.minW < this.column) { node.w = Math.max(node.w, node.minW); }

```

Comment:
I think we want node.minW <= this.column as minWidth:12 should force to take 12 column for example.